charts/dapr/charts/dapr_operator/templates/dapr_operator_deployment.yaml

```diff
@@ -208,12 +156,16 @@ spec:
 {{- end }}
       serviceAccountName: dapr-operator
       volumes:
-        - name: credentials
-          secret:
-            secretName: dapr-trust-bundle
-        - name: webhook-creds
-          secret:
-            secretName: dapr-webhook-cert
+        - name: dapr-trust-bundle
+          configMap:
+            name: dapr-trust-bundle
+        - name: dapr-identity-token
```

Comment:
Leaving a comment here as a reminder that this too needs (in a separate PR) to have an option to support non-projected volumes